render-ws-java-client/src/test/java/org/janelia/render/client/BlockLayoutCreatorTest.java

```diff
@@ -134,4 +133,40 @@ public void specifyingDimensionTwiceThrowsError() {
 			assertEquals("Intervals for dimension x already specified", e.getMessage());
 		}
 	}
+
+	@Test
+	public void sameMaxZForRegularAndShiftedGrids() {
```

Comment:
I think this should be renamed or split into two tests:
* the regular and shifted parts are independent,
* both could fail by having the same `maxZ` that is different from the one of `bounds`, which would not contradict the test name.

Reply:
sure - I split it into 2 tests